primefaces/src/main/resources/META-INF/resources/primefaces/datatable/datatable.js

```diff
@@ -809,6 +809,9 @@ PrimeFaces.widget.DataTable = PrimeFaces.widget.DeferredWidget.extend({
             if(this.cfg.rowSelectMode !== 'none') {
                 this.bindRowEvents();
             }
+            else {
+                this.jq.find('tr.ui-datatable-selectable').css('cursor', 'default');
```

Comment:
Why are there rows classed with `ui-datatable-selectable` when row select mode is `none`?

Reply:
Take a look at the initial PR #9430. It's a difference between selecting via row click vs. actually clicking the checkbox, but removing the style affects the checkboxes too.